lib/SetAPI/sampleset/SampleSetInterface.py

```diff
@@ -35,68 +35,63 @@ def create_sample_set(self, ctx, params):
                 item_condition, list
             ):  # Auto populate UI puts input into an array
                 if len(item_condition) > 1:
-                    err_msg = "ERROR: please only select 1 condition per Reads"
+                    err_msg = "ERROR: please only select 1 condition per reads object"
                     raise ValueError(err_msg)
                 item["condition"] = item_condition[0]
 
             params["sample_ids"].extend(item["sample_id"])
-            params["condition"].extend([item["condition"] for i in item["sample_id"]])
 
+            params["condition"].extend([item["condition"] for i in item["sample_id"]])
         pprint(params)
+        ### Create the working dir for the method; change it to a function call
+        out_obj = {k: v for k, v in params.items() if k not in ("ws_id",)}
```

Comment:
this was all in a massive try/except - changed the try so it only covers the part that is actually likely to fail!